frontend/webapp/utils/ldap/forms.py

```diff
@@ -0,0 +1,106 @@
+"""Provides Flask-Security login forms for usage with LDAP auth backend."""
+from flask import current_app
+from flask import request
+from flask_security.confirmable import requires_confirmation
+from flask_security.forms import Form
+from flask_security.forms import get_form_field_label
+from flask_security.forms import NextFormMixin
+from flask_security.utils import config_value
+from flask_security.utils import encrypt_password
+from flask_security.utils import get_message
+from flask_security.utils import verify_and_update_password
+from ldap3.core.exceptions import LDAPExceptionError
+from werkzeug.local import LocalProxy
+from wtforms import BooleanField
+from wtforms import PasswordField
+from wtforms import StringField
+from wtforms import SubmitField
+
+_datastore = LocalProxy(lambda: current_app.extensions["security"].datastore)
```

Comment:
is this datastore synced with the local one (frontend DB), or sth separate?

Reply:
If no, is it sth temporary? If yes, if someone is e.g., removed from the LDAP, is the local datastore updated? How does password change work with LDAP and our current implementation?